plugins/catalog/src/components/AboutCard/AboutCard.tsx

```diff
@@ -182,7 +184,13 @@ export function AboutCard({ variant }: AboutCardProps) {
           </>
         }
         subheader={
-          <HeaderIconLinkRow links={[viewInSource, viewInTechDocs, viewApi]} />
+          <HeaderIconLinkRow
+            links={
+              isComponent
```

Comment:
I think we could just reuse the existing `hasApis` here instead right?

Reply:
Depends. The question is whether we want to show the greyed-out button for APIs to hint the user that it is an option for components to have APIs. If we use `hasApi`, we always hide if there is no API relation. The "API" tab will be there, nevertheless, so it is just the question if we want to have this extra pointer on the `AboutCard` for components.

In my opinion, the "View API" button lost part of its usefulness when we moved away from showing the rendered API docs directly in the "API" tab and in contrast only show tables that link to API entity pages. But it is fine to keep it. If we decide to hide it, we can also delete the ["No APIs available" logic](https://github.com/backstage/backstage/pull/7886/files#diff-1488984aa6b8bbad3dcab3cd03465506902a52454c7a49eb119c4498365ad557R132) since it isn't used anymore.

Reply:
Yeah, we thought about this first, but that would make the `disabled` state obsolete. As an API mainly makes sense for a `component` kind, we wanted to exclude it from the other kinds.

What do you think?

Reply:
Yes that's correct; maybe it's best to remove it. This button confused some of our users because they expected that it would link to the definition tab on API entities, but it was just greyed out there. If someone makes an effort to make the `AboutCard` more configurable/composable in the future, we hopefully don't need to discuss these issues in the catalog plugin anymore.

Reply:
Thanks for the feedback @Rugvip & @dhenneke, according to your discussion I adjusted the PR and removed the icon completely. 👏  Could you double check if it looks good to you now?